content/modules/enterprise-capabilities.md

```diff
@@ -0,0 +1,84 @@
+| Module datatypes        | Yes        | Yes          | Yes        | Yes          |
+| Multikey methods        | Yes (v1.0) | No           | Yes (v1.0) | No           |
+| Persistence (AOF)       | Yes (v1.0) | Yes (v2.0)   | Yes (v1.0) | Yes (v2.0)   |
+| Persistence (snapshop)  | Yes (v1.0) | Yes (v2.0)   | Yes (v1.0) | Yes (v1.0)   |
+| Redis on Flash (RoF)    | No         | Yes (vTBD)   | Yes (vTBD) | No           |
+| Replica Of              | Yes (v1.0) | Yes (v2.0)   | No         | Yes (v2.2)   |
+| Reshard/rebalance       | No         | Yes (v2.0)   | Yes (v1.0) | No           |
+
+| Feature name/capability | [RedisJSON]({{< relref  "/modules/redisjson" >}})    | [RediSearch]({{< relref  "/modules/redisearch" >}})    | [RedisTimeSeries]({{< relref  "/modules/redistimeseries" >}}) |
+|-------------------------|:------------:|:-------------:|:---------------:|
+||||<br/>|
+| Supported by Redis Enterprise Software | Yes | Yes | Yes | 
+| Supported by Redis Enterprise Cloud    | Yes | Yes | Yes |
+||||<br/>|
+| Active-Active (CRDB)    | No           | Yes (v2.0)    | No           | 
+| Backup/Restore          | Yes (v1.0)   | Yes (v1.4)    | Yes (v1.2)   | 
+| Clustering              | Yes (v1.0)   | Yes (v1.6)    | Yes (v1.2)   | 
+| Custom hashing policy   | Yes (v1.0)   | Yes (v2.0)    | Yes (v1.2)   | 
+| Eviction expiration     | Yes (v1.0)   | Yes (v2.0)    | Yes (v1.2)   | 
+| Failover/migration      | Yes (v1.0)   | Yes (v1.4)    | Yes (v1.2)   | 
+| Internode encryption    | Yes (v1.0.8) | Yes (v2.0.11) | Yes (v1.4.9) | 
+| Module datatypes        | Yes          | Yes           | Yes          | 
+| Multikey methods        | No           | Yes (v2.0     | Yes (v1.2)   | 
+| Persistence (AOF)       | Yes (v1.0)   | Yes (v1.4)    | Yes (v1.2)   | 
+| Persistence (snapshop)  | Yes (v1.0)   | Yes (v1.6)    | Yes (v1.2)   | 
+| Redis on Flash (RoF)    | Yes (v1.0)   | Yes (v2.0)    | Yes (v1.4.7) | 
+| Replica Of              | Yes (v1.0)   | Yes (v1.6)    | Yes (v1.2)   | 
+| Reshard/rebalance       | Yes (v1.0)   | Yes (v2.0)    | Yes (v1.2)   | 
+
+## Feature descriptions
+
+The following table briefly describes each feature shown in the earlier tables.
+
+| Feature name/capability | Description |
+|-------------------------|-------------|
+| Active-Active (CRDB)    | Supports Active-active (CRDB) databases  |
+| Backup/Restore          | Supports import and export features |
+| Clustering              | Supports sharded databases and shards that can be migrated |
+| Custom hashing policy   | Supports databases with custom hashing policies |
+| Eviction expiration     | Fully supports eviction policies and expiration |
+| Failover/migration      | Supports databases in failover or migration status |
+| Internode encryption    | Supports encryption on the data plane, when enabled for databases |
+| Module datatypes        | Supports module-specific datatypes and existing Redis types |
```

Comment:
| Eviction expiration     | Fully supports eviction policies and expiration |
 
This means that data can be evicted by Redis Enterprise when the database reaches limits.  This used not to be possible with RediSearch because you could evict parts of the index instead of the data

| Failover/migration      | Supports databases in failover or migration status |

Supports the failover between Primary and replica but more importantly the migration of shards between nodes within the Redis Enterprise cluster

| Module datatypes        | Supports module-specific datatypes and existing Redis types |

Please remove from the table